auth-api/src/auth_api/services/authorization.py

```diff
@@ -154,9 +154,14 @@ def check_auth(token_info: Dict, **kwargs):
         if business_identifier:
             auth = Authorization.get_user_authorizations_for_entity(token_info, business_identifier)
         elif org_identifier:
-            auth_record = AuthorizationView.find_user_authorization_by_org_id(token_info.get('sub', None),
-                                                                              org_identifier)
-            auth = Authorization(auth_record).as_dict() if auth_record else None
+            product_code_in_jwt = token_info.get('product_code', None)
```

Comment:
For a user token product_code won't be present in the token. Only for service account token we can map the product code as a claim. The auth check for product code should be done only for service account tokens and user token auth check needs to be left as is.

Reply:
Fixed. System role will check product code. Add the product subscription when create a org.